app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(players: list) -> None:
+    return sum(player.get_rating() for player in players)
+
+
+def elves_concert(players: list) -> None:
+    [player.play_elf_song() for player in players]
+
+
+def feast_of_the_dwarves(players: list) -> None:
```

Comment:
dwarves, I guess..
Also, make more descriptive type hints, for example, list[<class_name>]. Fix everywhere